Command/IndexImportCommand.php

```diff
@@ -53,13 +59,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $manager = $this->getManager($input->getOption('manager'));
 
+        $options = [];
+        if ($input->getOption('gzip')) {
+            $options['gzip'] = true;
```

Comment:
You should set it to `null` with current implementation. With `true`/`false` it behaves the same, as with any other value.